src/azure-cli/azure/cli/command_modules/appconfig/_help.py

```diff
@@ -169,8 +171,10 @@
 type: command
 short-summary: Lists revision history of key-values.
 examples:
-  - name: List revision history of key "color" label "test" using App Configuration name.
+  - name: List revision history of a key-value using App Configuration name.
     text: az appconfig revision list -n MyAppConfiguration --key color --label test
+  - name: List revision history of a key-value with multiple labels.
+    text: az appconfig revision list -n MyAppConfiguration --key color --label test,prod,\\0
```

Comment:
Can you confirm if \\0 works well in all clients including Powershell, cmd, bash and so on?

Reply:
Yes, it works fine in these command shell. Quotes are needed when input arguments contain spaces.